packages/slice-machine/lib/builders/common/EditModal/index.js

```diff
@@ -164,8 +164,8 @@ const EditModal = ({ close, data, fields, onSave, getFieldMockConfig }) => {
             return null;
           })();
 
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-          const updatedValue = deepMerge(initialModelValues, value);
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-assignment
+          const updatedValue = { ...initialModelValues, ...value };
```

Comment:
Do we need the deepMerge function anymore? i believe it was only used here right?

Reply:
it's also used on line 106 :)